### PR TITLE
Concrete manifest example

### PR DESCRIPTION
**Pull request by asselitx**
Also explain deploy-vars better in the README, and add a note to
the scratchpad.

Signed-off-by: Terrence Asselin <terrence.asselin@lexisnexisrisk.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,11 @@ This way you can organize scripts in 'shared' files where different subsets of t
 
 ### Deployment Variables
 
-As Tony pointed out the variable notation must be unique for each level of our scripts that does variable replacement. For deployment variables, ensure that the **%xxx%** notation that won't conflict with any other type of escaping or variable notation. 
+Deployment variables are in the provisional format **%variablename%**. We'll verify that this notaion doesn't conflict with any notation or escaping used at any other levels of scripting- including URL and XML escaping, XSLT, XML and our own scripting language.
```
Comment (timothyklemm) [inline README.md:75]: Typo: notation, not notaion.

File: README.md [hunk 2]
```diff
@@ -72,7 +72,11 @@ This way you can organize scripts in 'shared' files where different subsets of t
...
+
+These variables are for the purpose of customizing a final output-binding-config for different deployment environments such as QA, Development and Production. Example uses include different URLs, usernames and passwords required for the different environments. For a given release, an output-binding-config will be identical for all environments with the exception of these values.
```
Comment (timothyklemm) [inline README.md:77]: Last sentence... Should "will be" be replaced with something less absolute? Thinking long term, if we ever support something like the test repo is it possible that we would want to support configuration that only exists in specific environments (like the test repo is never intended to be configured in Prod or DR)?

Not suggesting it be defined now, but maybe this verbiage is a bit too strict.

File: README.md [hunk 3]
```diff
@@ -72,7 +72,11 @@ This way you can organize scripts in 'shared' files where different subsets of t
...
+
+Variable replacement is a simple replacement of the **%variablename%** markup with the corresponding value found in the deploy-vars.xml file. This replacement is done by the Build Tool as it is building the output-binding-config. No further processing of deploy-vars will need to be done during script loading or script runtime.
```
Comment (timothyklemm) [inline README.md:79]: Can a variable value be defined with references to other variables?

**Comment by asselitx** [comment]
outdated, don't need to keep

**Pull request closed without merging**